### PR TITLE
[HttpClient] fix timing measurements with NativeHttpClient

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q             | A
| ------------- | ---
| Branch?       | 4.3
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | -
| License       | MIT
| Doc PR        | -

Align to what curl does.

### CODE REVIEW

File: src/Symfony/Component/HttpClient/NativeHttpClient.php [hunk 1]
```diff
@@ -133,14 +133,14 @@ public function request(string $method, string $url, array $options = []): Respo
 
         // Always register a notification callback to compute live stats about the response
         $notification = static function (int $code, int $severity, ?string $msg, int $msgCode, int $dlNow, int $dlSize) use ($onProgress, &$info) {
-            $now = microtime(true);
-            $info['total_time'] = $now - $info['start_time'];
+            $info['total_time'] = microtime(true) - $info['start_time'];
```
Comment (OskarStark) [inline src/Symfony/Component/HttpClient/NativeHttpClient.php]: Just for information, why not using the stopwatch here?

Comment (nicolas-grekas) [reply src/Symfony/Component/HttpClient/NativeHttpClient.php]: "Why use it?" would my answer :)
More seriously, this backports something that is provided by curl natively. We don't want nor need to provide Stopwatch to curl when it already does everything without us doing anything. This provides "same behavior" to NativeHttpClient, to make them as alike as possible.

Comment (OskarStark) [reply src/Symfony/Component/HttpClient/NativeHttpClient.php]: Thanks

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @nicolas-grekas.

**Pull request merged**